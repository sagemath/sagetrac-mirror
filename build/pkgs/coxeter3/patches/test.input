type
A
3
compute
1 3 2 1 2 3 1 2 1
ihbetti
213
q
q
