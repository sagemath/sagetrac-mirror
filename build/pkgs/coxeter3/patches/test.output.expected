This is Coxeter version 3.1.sage.
Enter help if you need assistance, carriage return to start the program.

coxeter : 
type : 
rank : coxeter : enter your element (finish with a carriage return) :
213
coxeter : enter your element (finish with a carriage return) :
h[0] = 1  h[1] = 3  h[2] = 3  h[3] = 1  

size : 8

coxeter : coxeter : 